### PR TITLE
SOMMELIER-19

### PR DESCRIPTION
**Pull request by constantkek**

### CODE REVIEW

**Comment by constantkek** [comment]
checked:
image
unchecked:
image

**Review by Mixxx-a — APPROVED** [review]
Not sure if everything is correct, but i'll add tests to it

File: App/components/RadioButton/RadioButton.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+type Props = {
+  checked?: boolean
+  children?: ReactNode
+  onPress?: () => void
+}
+
+const RadioButton = ({
+  checked = true,
+  children = 'Red wine',
```
Comment (Mixxx-a) [inline App/components/RadioButton/RadioButton.tsx]: Not sure if setting default children like that is a good idea

**Review by Mixxx-a — APPROVED** [review]

File: App/components/RadioButton/RadioButton.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    <RadioButtonContainer>
+      <TouchableOpacity style={styles.circle} onPress={onPress}>
+        {checked ? <View style={styles.checkedCircle} /> : <View />}
+      </TouchableOpacity>
+      <RadioButtonText>{children}</RadioButtonText>
+    </RadioButtonContainer>
+  )
+}
+
+const styles = StyleSheet.create({
```
Comment (StanislavMyakishev) [inline App/components/RadioButton/RadioButton.tsx]: Maybe this functionality should be implemented via styled components? 
```javascript
const CheckedViewCircle = styled(View)`
// code from stylesheet circle
`
```

**Comment by constantkek** [comment]
@StanislavMyakishev works 100% (on android ofc)

File: App/components/RadioButton/RadioButton.styled.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  borderRadius: 10
+  borderWidth: 1
+  borderColor: #757575
+  alignItems: center
+  justifyContent: center
+  marginRight: 40
+`
+
+export const RadioButtonViewFilled = styled.View`
+  width: 20
```
Comment (StanislavMyakishev) [inline App/components/RadioButton/RadioButton.styled.ts]: Are you sure that those styled work just fine? :)

Comment (constantkek) [reply App/components/RadioButton/RadioButton.styled.ts]: @StanislavMyakishev that works. We extend filled circle to unfilled border. Thats either genius or костыль)))), but it solves the problem

File: App/components/RadioButton/RadioButton.styled.ts [hunk 4]
```diff
@@ -9,3 +10,25 @@ export const RadioButtonContainer = styled.View`
 export const RadioButtonText = styled.Text`
   color: black;
 `
+
+export const RadioButtonViewUnfilled = styled.View`
```
Comment (Maximus135) [inline App/components/RadioButton/RadioButton.styled.ts:13]: Why we can't use only one Radio Button View with rather styles depends on props?
As far as I can judge this is a duplication of code)

**Pull request merged**